### PR TITLE
Post-docker installation steps, including Windows deviating steps

### PR DESCRIPTION
**Pull request by cansav09**
In response to Issue #20, this PR addresses updates to the README.md file according to the steps @jaclyn-taroni laid out in that issue. 
This attempts to have step by step instructions for Windows and Mac computers. However, I suspect this may change depending on how the dry run tomorrow goes and if we encounter further glitches with Windows computers. 
More screenshots might be good.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,57 @@ This module focuses on the analysis of RNA-seq data.
...
+  the `IP & PORTS` pane. Copy this URL and paste it into the navigation bar of
+  your web browser.
+
+
+6. Log into `RStudio`. The username will be 'rstudio' and the password will be
+whatever you selected above (can also be accessed from the `Settings` >
+`General` panel).
+
+
+7. You should see a kitematic/ folder in your `RStudio` Files panel. Click on it.
```
Comment (jaclyn-taroni) [inline README.md]:
```suggestion
7. You should see a `kitematic/` folder in your `RStudio` Files panel. Click on it.
```

File: README.md [hunk 2]
```diff
@@ -6,16 +6,57 @@ This module focuses on the analysis of RNA-seq data.
...
+
+
+### If Kitematic doesn't work:
+If all else fails and Kitematic is not working for you, go to your Terminal or
+Command Prompt (for Mac or Windows respectively) and type in the following, but
+replacing <PATH_TO_TRAINING_FOLDERS> bit with the absolute path to
+"main directory" that was transferred from the flash/hard drive.
+```
+docker run -it --rm --mount type=volume,dst=/home/rstudio/kitematic,volume-driver=local,volume-opt=type=none,volume-opt=o=bind,volume-opt=device=<PATH_TO_TRAINING_FOLDERS> -e PASSWORD=<PASSWORD> -p 8787:8787
+```
```
Comment (jaclyn-taroni) [inline README.md]: This doesn't cover the getting to RStudio in your browser step.

File: README.md [hunk 3]
```diff
@@ -6,16 +6,57 @@ This module focuses on the analysis of RNA-seq data.
...
+`General` panel).
+
+
+7. You should see a kitematic/ folder in your `RStudio` Files panel. Click on it.
+If you do not see the training modules folders in the kitematic folder, raise
+your hand.
+
+
+### If Kitematic doesn't work:
+If all else fails and Kitematic is not working for you, go to your Terminal or
```
Comment (jaclyn-taroni) [inline README.md]: Forgot to note that I would add a line between the header and this text

File: README.md (hunk 2 above)
Comment (cansav09) [reply README.md]: Oh. I guess I should just put that they can follow the other steps the same way. (Although I'm just presuming that with the Windows).

**Comment by cansav09** [comment]
Okay. Feel free to take a gander when you can.

File: README.md [hunk 4]
```diff
@@ -6,16 +6,66 @@ This module focuses on the analysis of RNA-seq data.
...
+
+- In *Mac*, search for and open `Terminal`.
+- In *Windows*, search for and open `Command Prompt`.
+
+  In your respective command line interface, copy and paste the following:
+```
+docker pull ccdl/training_rnaseq:2019-houston
+```
+
+2. Run the container. Change the PASSWORD in the line below to whatever you'd
```
Comment (jaclyn-taroni) [inline README.md]: Which PASSWORD? Probably better to say
```suggestion
2. Run the container. Change the `<PASSWORD>` in the line below to whatever you'd
```

File: README.md [hunk 5]
```diff
@@ -6,16 +6,66 @@ This module focuses on the analysis of RNA-seq data.
...
+
+4. `Settings` > `Volumes` > Set local folder to "main directory" that was
+transferred from the flash/hard drive, using the `CHANGE` button.
 ![Folder](screenshots/all-02-volume.png)
-6. On the advanced panel you should that a TTY is assigned and STDIN will remain open. 
-![Advanced](screenshots/all-03-advanced.png)
-7. Visit the IP address that was listed in the PUBLISHED IP tab in a web browser and you should see an RStudio login. The login information is username: `rstudio` and password: `rstudio`. If you do not see this, raise your hand. 
-![RStudio](screenshots/all-04-rstudio.png)
+
+
```
Comment (jaclyn-taroni) [inline README.md]: Extra line?

File: README.md [hunk 6]
```diff
@@ -6,16 +6,66 @@ This module focuses on the analysis of RNA-seq data.
...
+2. Run the container. Change the PASSWORD in the line below to whatever you'd
+  like.
+```
+docker run -e PASSWORD=<PASSWORD> -p 8787:8787 ccdl/training_rnaseq:2019-houston
+```
+
+3. Open `Kitematic` - you should see an image running.
+
+
```
Comment (jaclyn-taroni) [inline README.md]: Extra line?

File: README.md [hunk 7]
```diff
@@ -6,16 +6,66 @@ This module focuses on the analysis of RNA-seq data.
...
+![Folder](screenshots/all-01-network.png)
+
+  - Alternatively, for a *Mac*, you can navigate to the RStudio window by typing
+    `localhost:8787` in your web browser
+
+6. Log into `RStudio`. The username will be 'rstudio' and the password will be
+whatever you selected above (can also be accessed from the `Settings` >
+`General` panel).
+
+
```
Comment (jaclyn-taroni) [inline README.md]: Extra line?

File: README.md [hunk 8]
```diff
@@ -6,16 +6,66 @@ This module focuses on the analysis of RNA-seq data.
...
+`Command Prompt` (for Mac or Windows respectively) and type in the following, but
+replacing <PATH_TO_TRAINING_FOLDERS> bit with the absolute path to
+"main directory" that was transferred from the flash/hard drive.
+```
+docker run -it --rm --mount type=volume,dst=/home/rstudio/kitematic,volume-driver=local,volume-opt=type=none,volume-opt=o=bind,volume-opt=device=<PATH_TO_TRAINING_FOLDERS> -e PASSWORD=<PASSWORD> -p 8787:8787
+```
+After starting your container this way, you can get to the RStudio window in
+a similar way as described above:
+- In Mac, type: `localhost:8787` in your web browser.
+- In Windows, go to `Command Prompt`, type: `ipconfig` and click enter.
```
Comment (jaclyn-taroni) [inline README.md]: 👍 we definitely have to test this on Windows 10 Pro

Comment (cansav09) [reply README.md]: For sure.

**Pull request merged**